tests/devlxd-container

```diff
@@ -114,7 +114,9 @@ kill -9 "${monitorPID}"
 
 # If the guest retrieved the image from the host, the host should emit an "image-retrieved" lifecycle event. The
 # requestor address tells us that this was definitely the guest.
-[ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.address')" = "@devlxd" ]
+# Newer LXD versions use the protocol instead of the requestor address to distinguish devlxd requests
+[ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.address')" = "@devlxd" ] ||
+[ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.protocol')" = "devlxd" ]
```

Comment:
Please indent the second line a bit more to make it clear/obvious that it's a continuation of the previous check. Thanks!

Reply:
I believe you need a ` \` at the end of the first line for it to be continued to the next one.